src/main/java/uk/gov/ons/census/action/messaging/FulfilmentRequestReceiver.java

```diff
@@ -25,7 +25,11 @@
               "P_OR_I4",
               "P_UAC_UACIP1",
               "P_UAC_UACIP2B",
-              "P_UAC_UACIP4"));
+              "P_UAC_UACIP4",
```

Comment:
Per @AdamHawtin 's comments - we only need to add things to this list if we are waiting for an HI case to be created by the Case Processor, because we intend to hang UAC-QID pairs off it. Because we don't need a UAC on any large print fulfilments for individuals, we can go ahead and fulfil these immediately (provided we ignore them in Case Processor by not adding the types to the list that Adam already identified)